source/tutorials/test-a-react-todo-app/react-todo-item-testing.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 5. Testing todo item behavior
+comments: false
+layout: video
+containerClass: tutorial
+---
+
+{% video 242954792 %}
+
+## What's covered
+
+We will update our app to properly display todo items based on their `isComplete` property, adding tests to verify the proper behavior as we go. From there, we'll test and implement the item deletion functionality. We will cover interacting with an element that is hidden unless hovered over and look at different ways of handling this situation. We'll also look at the appropriate way to hold onto references to previously queried DOM elements using {% url `as()` as %} to create aliases.
```

Comment:
alias command should be written as `.as()` for consistency

Reply:
good catch!